components/dashboard/src/components/podkit/combobox/Combobox.tsx

```diff
@@ -310,9 +309,9 @@ type ComboboxItemProps = {
 };
 
 export const ComboboxItem: FC<ComboboxItemProps> = ({ element, isActive, onSelected, onFocused }) => {
-    let selectionClasses = `dark:bg-gray-800 cursor-pointer`;
+    let selectionClasses = `bg-pk-surface-tertiary/25 cursor-pointer`;
     if (isActive) {
-        selectionClasses = `bg-gray-200 dark:bg-gray-700 cursor-pointer focus:outline-none focus:ring-0`;
+        selectionClasses = `bg-pk-content-tertiary/10 cursor-pointer focus:outline-none focus:ring-0`;
```

Comment:
yah, I'm trying to use the surface colors from podkit, but have noticed they don't all work together w/ the right contrast. When you say a slightly lighter background, do you mean the main container background?

Reply:
Lightening up the surface there starts to make the rest of the light colors not have enough contrast either. I agree this doesn't feel great though. I might call this one good since it's pretty close to what's live today, and it's probably worth having design take a stab at this combobox component and revisit it.